syntax = "proto3";

package test;

message ChunkEdges {
  message Edges {
    bytes edgeList = 1;
    bytes affinities = 2;
    bytes areas = 3;
  }

  Edges inChunk = 4;
  bytes crossChunk = 5;
  Edges betweenChunk = 6;
}